src/EnergyPlus/FaultsManager.cc

```diff
@@ -1764,6 +1764,102 @@ namespace FaultsManager {
             FouledCoils(jFault_FoulingCoil).Rfa = rNumericArgs(3);
             FouledCoils(jFault_FoulingCoil).Aout = rNumericArgs(4);
             FouledCoils(jFault_FoulingCoil).Aratio = rNumericArgs(5);
+
+
+            // Coil check and link
+            {
+                // Obtains and Allocates WaterCoil related parameters from input file
+                if (WaterCoils::GetWaterCoilsInputFlag) {
+                    WaterCoils::GetWaterCoilInput();
+                    WaterCoils::GetWaterCoilsInputFlag = false;
+                }
+
+                // Check the coil name and type
+                int CoilNum = UtilityRoutines::FindItemInList(FouledCoils(jFault_FoulingCoil).FouledCoilName, WaterCoils::WaterCoil);
+                if (CoilNum <= 0) {
+                    ShowSevereError(cFaultCurrentObject + " = \"" + cAlphaArgs(1) + "\". Referenced Coil named \"" +
+                                    FouledCoils(jFault_FoulingCoil).FouledCoilName + "\" was not found.");
+                    ErrorsFound = true;
+                } else {
+                    // Coil is found: check if the right type
+                    if ( (WaterCoils::WaterCoil(CoilNum).WaterCoilType_Num == WaterCoils::WaterCoil_SimpleHeating) ||
+                         (WaterCoils::WaterCoil(CoilNum).WaterCoilType_Num == WaterCoils::WaterCoil_Cooling) )
+                    {
+                        // Link the Coil with the fault model
+                        WaterCoils::WaterCoil(CoilNum).FaultyCoilFoulingFlag = true;
+                        WaterCoils::WaterCoil(CoilNum).FaultyCoilFoulingIndex = jFault_FoulingCoil;
+
+                        FouledCoils(jFault_FoulingCoil).FouledCoiledType = WaterCoils::WaterCoil(CoilNum).WaterCoilType_Num;
+                        FouledCoils(jFault_FoulingCoil).FouledCoilNum = CoilNum;
+
+                        SetupOutputVariable("Coil Fouling Factor",
+                                    OutputProcessor::Unit::K_W,
+                                    WaterCoils::WaterCoil(CoilNum).FaultyCoilFoulingFactor,
+                                    "System",
+                                    "Average",
+                                    WaterCoils::WaterCoil(CoilNum).Name);
+
+                        // Coil:Cooling:Water doesn't report UA because it's not variable,
+                        // but here, it's useful since we do change it via fouling, so report it
+                        if (WaterCoils::WaterCoil(CoilNum).WaterCoilType_Num == WaterCoils::WaterCoil_Cooling) {
+                            SetupOutputVariable("Cooling Coil Total U Factor Times Area Value",
```

Comment:
@mjwitte can you use your eagle eye on these output variable names?  Everything looks fine to me, but a second look would be good.

Reply:
@Myoldmopar These output variables are all good, the "U Factor Times Area Value" is consistent with the existing "Heating Coil U Factor Times Area Value" output and with the sizing report for other water coils.